src/mscorlib/shared/System/ValueTuple.cs

```diff
@@ -470,13 +444,16 @@ string IValueTupleInternal.ToStringEnd()
     public struct ValueTuple<T1, T2>
         : IEquatable<ValueTuple<T1, T2>>, IStructuralEquatable, IStructuralComparable, IComparable, IComparable<ValueTuple<T1, T2>>, IValueTupleInternal, ITuple
     {
+        private static readonly EqualityComparer<T1> s_t1Comparer = EqualityComparer<T1>.Default;
```

Comment:
Why did you make this change?

Reply:
ValueTuple in CoreCLR is optimized for current RyuJIT. 

ValueTuple in CoreFX is for older runtimes (e.g. .NET Framework 4.0). It is optimized for JITs that shipped in these older runtimes. 

The two are different by design. It is not desirable to unify them.